packages/terra-demographics-banner/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "terra-demographics-banner",
+  "main": "lib/DemographicsBanner.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-demographics-banner",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-demographics-banner",
+    "DemographicsBanner",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-props-table": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-intl": "^2.2.3",
+    "shortid": "^2.2.8",
+    "terra-i18n": "^0.0.0",
```

Comment:
npm does not treat matching of `^0.0.0` and `^1.0.0` versions the same when using the caret character.

This can be tested with https://semver.npmjs.com/.

Using the caret will update the package to the most recent major version. `^1.2.3` will match any `1.x.x` release matching version `1.2.3` or higher including `1.2.4`, `1.3.0`, `1.40.1`, but will hold off on `2.0.0`.

Though using the caret with a `0` based major version, the caret doesn't have any impact.

npm will only match `^0.0.0` to version `0.0.0`. 
npm will only match `^0.1.0` to version `0.1.0`.

To match all minor and patch releases under a `0` major release there are a couple of options. We can use `0.x` or `<1.0.0`.

We've been using `"^0.x"` for components on the `0` major version.

**tl;dr Can you update the terra-i18n,  terra-icon, and terra-toolkit packages to reference** `"^0.x"`